Formula/pdmenu.rb

```diff
@@ -0,0 +1,31 @@
+class Pdmenu < Formula
+  desc "Full screen menuing system for Unix"
+  homepage "https://joeyh.name/code/pdmenu/"
+  url "http://sources.buildroot.net/pdmenu/pdmenu-1.3.6.tar.gz"
+  sha256 "dedd2a4a5719278b5e49041161990c2f20b5909818837542aaca01819f2c14eb"
+  license "GPL-2.0-only"
+
+  depends_on "coreutils"
```

Comment:
Does it need this at runtime?

Reply:
Yes, the install statements in Makefile fail with macOS /usr/bin/install (on my Mac) but succeed with GNU flavor, ginstall. That's why I had to put this dependency.

Reply:
But the makefile isn't needed when you run the program, only when you install it right?

Reply:
Yes, you're right. I don't know exactly how to use `ginstall` instead `/usr/bin/install` during the build process.

```ruby
depends_on "coreutils" => :build 
```

?

Reply:
If it's only needed in the build, `depends_on "coreutils" => :build ` is the correct way.

Reply:
done